incubator/sparkoperator/templates/spark-rbac.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (.Values.rbac.create) (ne .Values.sparkJobNamespace "") }}
```

Comment:
I think this change should be kept, i.e., skipping creating RBAC if the namespace value is empty.

Reply:
I think It is not documented and not obvious.
In order to skip RBAC creation user will have to write.
```
helm install incubator/sparkoperator --namespace spark-operator --set sparkJobNamespace=
```
I suggest that adding a separate flag for enabling/disabling RBAC creation is better.
Any way I can revert it.
Should I do it?

Reply:
I think we can fix the documentation instead.

Reply:
That's fine with me.

Reply:
@MelnykAndriy can you update the documentation? We can get this merged after that. Thanks!